src/Mvc/Mvc.Razor/src/MvcRazorLoggerExtensions.cs

```diff
@@ -1,204 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Diagnostics;
-using Microsoft.Extensions.Logging;
-
-namespace Microsoft.AspNetCore.Mvc.Razor;
```

Comment:
Nearly all of the log messages on this types, except the one you see in the previous file, had been moved to individual types over time, and hence weren't referenced.